src/main/java/org/folio/bulkops/controller/BulkOperationController.java

```diff
@@ -84,6 +84,7 @@ public ResponseEntity<UnifiedTable> getPreviewByOperationId(UUID operationId, Bu
 
   @Override
   public ResponseEntity<BulkOperationRuleCollection> postContentUpdates(UUID operationId, BulkOperationRuleCollection bulkOperationRuleCollection) {
+    bulkOperationService.deleteIfExists(operationId);
```

Comment:
@AntonAntonich , the next line will throw exception if operation will be removed/

Reply:
not actual